client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -109,7 +89,7 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
 
           <Box textAlign="center" marginTop={5}>
             <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
-              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Create'}
+              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'SIGN UP'}
```

Comment:
Please avoid the usage of CAPITAL CASE text for accessibility reasons - use a CSS class to transform text instead